policy/datasets.yaml

```diff
@@ -1398,17 +1398,10 @@ measurePhotonTransferCurve_metadata:
     storage: YamlStorage
     python: lsst.daf.base.PropertySet
     template: ''
-measurePhotonTransferCurveGainAndNoise:
-    description: "Persisted output data (gain and noise) from the MeasurePhotonTransferCurveTask"
-    persistable: ignored
-    python: lsst.cp.pipe.ptc.PtcDataset
-    storage: PickleStorage
-    tables: 'raw'
-    template: ''
-measurePhotonTransferCurveDatasetAll:
+photonTransferCurveDataset:
     description: "Persisted output data from the MeasurePhotonTransferCurveTask"
     persistable: ignored
```

Comment:
What does this "persistable" option mean and why is it set to "ignored"? The description says that you are persisting data.

Reply:
See OOB communication "explaining" this (in the loosest possible sense of "explain").